hack/testing/lib/test-functions

```diff
@@ -7,3 +7,40 @@ function announce_test(){
   echo "     $message" 
   echo $TEST_DIVIDER
 }
+
+function start_logging_builds()
+{
+  logging_builds=$(oc get bc -l logging-infra=development -o name)
+  for bc in $logging_builds ; do
+    if [ "${USE_LOCAL_SOURCE:false}" = false ] ; then
+      oc start-build $bc
+    else
+      oc start-build --from-dir $OS_O_A_L_DIR $bc
+    fi
+  done
+}
+
+function wait_for_logging_builds_to_complete()
+{
+    waittime=1800 # seconds - 30 min
+    interval=30
+    complete=0
+    while [ $waittime -gt 0 -a $complete = 0 ] ; do
+      buildstatus=$(oc get builds -l logging-infra=development -o jsonpath={.items[*].spec.phase})
+      complete=$(echo $buildstatus | grep -io "complete" | wc -l)
```

Comment:
`complete` here is a boolean - it should be `0` until _all_ of the builds have completed.  That's what the original code is doing:
```
        # all lines must have $4 == "Complete"
        complete=`oc get builds | awk '$4 == "STATUS" || $4 == "Complete" {complete++}; END {print NR == complete}'`
```
That is, see if _all_ builds are in the `Complete` state.